src/MessageContainer.js

```diff
@@ -109,13 +109,15 @@ export default class MessageContainer extends React.Component {
       message.user = {};
     }
 
+    let position = message.user._id === this.props.user._id ? 'right' : 'left';
```

Comment:
It seem to me that this could be improved using an _util_ function. Since if message is from system `user` should be null, you can write a function that receive a message and the current user  such as

```javascript

function messagePosition(message, currentUser) {
  if (message.fromSystem) {
    return 'center';
  }

  return (message.user._id === currentUser._id) ? 'right' : 'left';
}
```